charts/cluster-autoscaler/templates/_helpers.tpl

```diff
@@ -109,21 +109,52 @@ Return true if the priority expander is enabled
 {{- end -}}
 
 {{/*
-Return the autodiscoveryparameters for clusterapi.
+autoDiscovery.clusterName for clusterapi.
```

Comment:
Sorry for the gnarly diff in this file. Essentially with the introduction of `namespace` to the combinatorics below, it made sense to decompose this a bit. Not a non-horrible way to do this using go templates, at least I couldn't figure out one.

Reply:
Yeah, I don't think I've ever seen a nice way to do this, but this is at least an easier to grok end result.